crates/sui-framework/sources/governance/staking_pool.move

```diff
@@ -488,6 +488,10 @@ module sui::staking_pool {
 
     // ==== test-related functions ====
 
+    public fun validator_address(staked_sui: &StakedSui): address {
```

Comment:
Is this used anywhere? We may remove the field `validator_address` later (see the TODO on StakedSui struct) so it's better to not have code that depends on this field.